controllers/dspipeline_controller.go

```diff
@@ -177,7 +177,8 @@ func (r *DSPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	dspa := &dspav1alpha1.DataSciencePipelinesApplication{}
 	err := r.Get(ctx, req.NamespacedName, dspa)
 	if err != nil && apierrs.IsNotFound(err) {
-		log.Info("DSPA resource was not found")
+		// TODO change this to a Debug message when doing https://issues.redhat.com/browse/RHOAIENG-1650
+		log.Info("DSPA resource was not found, assuming it was recently deleted, nothing to do here")
```

Comment:
nitpick:  small grammar update (line seems like a run-on phrase).  would update to something like
`DSPA resource was not found - assuming it was recently deleted, nothing to do here`
or
`DSPA resource was not found, assuming it was recently deleted. Nothing to do here.`